src/v0/destinations/af/deleteUsers.js

```diff
@@ -57,7 +57,7 @@ const userDeletionHandler = async (userAttributes, config) => {
   if (config.statusCallbackUrls) {
     const statusCallbackUrlsArray = config.statusCallbackUrls.split(',');
     const filteredStatusCallbackUrlsArray = statusCallbackUrlsArray.filter((statusCallbackUrl) => {
-      const URLRegex = /^(https?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w!#$&'()*+,/:;=?@[\]~-]+$/;
+      const URLRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w!#$&'()*+,./:;=?@[\]~-]+$/;
```

Comment:
can you check all the similar regex patterns that got updated in other files too ?

Reply:
Yeah already checking